<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1557</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="digests_table">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>61</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Digest</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="actors_table">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>141</width>
      <height>430</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Actors</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="calendars_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>171</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>SYSTEM Stories</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="depotlinks_table">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>191</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Calendarlinks</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Weight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="system_delete_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>delete </string>
    </property>
   </widget>
   <widget class="QPushButton" name="system_insert_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add new System</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="system_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="system_update_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Change Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calendar_update_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Change Calendar Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calendar_insert_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add new Calendar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calendar_delete_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="calendar_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="actor_update_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Change Actor Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="actor_insert_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add new Actor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="actor_delete_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>delete Actor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="actor_name">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="depotlink_delete_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>delete CalendarLink</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="depotlink_name">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="depotlink_insert_button">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add new CalendarLink</string>
    </property>
   </widget>
   <widget class="QPushButton" name="depotlink_update_button">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update CalendarLink</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_all_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calendar_edit_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit </string>
    </property>
   </widget>
   <widget class="QPushButton" name="actor_edit_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit Actor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="depotlink_edit_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit CalendarLink</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="w_agenda_table">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>260</y>
      <width>371</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Actors Agenda</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="w_members_table">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>171</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Actors Members</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="w_groups_table">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>260</y>
      <width>171</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Actors Groups</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="w_ideas_table">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>391</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Actor Ideas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Idea walk</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="w_acptfacts_table">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>10</y>
      <width>371</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>AcptFacts Base</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>AcptFacts AcptFact</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="system_name_combo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="system_copy_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QComboBox" name="depotlink_type_combo">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>110</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="system_load_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="depotlink_weight">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calendarlink Weight:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="ignores_table">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>61</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Ignore</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="show_digests_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>digest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_ignores_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ignore</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_ignore_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>o</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_ignore_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_public_calendar_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>640</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>output_calendar to public_dir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reload_all_src_calendars_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>620</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>refresh_all_src_calendars</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_public_and_reload_srcs_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>660</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>dest2public and refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="isol_open_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>isol</string>
    </property>
   </widget>
   <widget class="QPushButton" name="isol_save_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="five_issue_button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5 issue</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1557</width>
     <height>21</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>File</string>
   </property>
   <property name="accessibleDescription">
    <string>General Operations</string>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="fm_new"/>
    <addaction name="fm_open"/>
    <addaction name="fm_save"/>
    <addaction name="save_as"/>
   </widget>
   <addaction name="file_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="fm_open">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="fm_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="save_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="fm_new">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
